wikidatarefisland/pipes/item_mapping_matcher_pipe.py

```diff
@@ -0,0 +1,48 @@
+from wikidatarefisland.pipes import AbstractPipe
+
+
+class ItemMappingMatcherPipe(AbstractPipe):
+    """A pipe segment to match potential references with statement data."""
+
+    def __init__(self, mapping, whitelisted_external_identifiers):
+        """Instantiate the pipe
+
+        Arguments:
+            matchers {wikidatarefisland.data_model.ValueMatchers} --
+                A static class with value matcher functions
+        """
```

Comment:
Needs to be removed, this is not a parameter of this constructor

Reply:
ken. It's just copy pasta error.